python/cudf/cudf/pandas/__init__.py

```diff
@@ -18,6 +18,25 @@ def install():
     loader = ModuleAccelerator.install("pandas", "cudf", "pandas")
     global LOADED
     LOADED = loader is not None
+    import os
+
+    cudf_pandas_mr = os.getenv("CUDF_PANDAS_MEMORY_RESOURCE", None)
+    if cudf_pandas_mr is not None:
+        import rmm
+        import rmm.mr
```

Comment:
I think you only need the second one:
```suggestion
        import rmm.mr
```